rpcd/playbooks/repo-fetcher.yml

```diff
@@ -0,0 +1,67 @@
+---
+- name: lay the repos down
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  user: root
+  tasks:
+    - name: lay the repos down
+      git:
+        dest: "{{ item.value.dest }}"
+        repo: "{{ item.value.repo }}"
+        version: "{{ item.value.version }}"
+      with_dict: "{{repos}}"
+    - name: generate rpcd ansible.cfg
+      copy:
+        dest: /opt/rpc-openstack/rpcd/playbooks/ansible.cfg
```

Comment:
We supply rpc-openstack/rpcd/playbooks/ansible.cfg -- should that get removed from the repo?